.github/workflows/check-for-spammy-prs.yml

```
@@ -0,0 +1,45 @@
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Save output to a file and then to environment
+        run: |
+          echo "${{ steps.pr_details.outputs.data }}" > pr_details.json
+          echo "files_count=$(jq --arg pr_details.json)" >> $GITHUB_ENV
+
+      - name: Add invalid label to PR if no files are changed
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: github.event.pull_request.title == 'Revert "Repo sync"' || env.files_count.changed_files == '0'
```

actionlint (1.7.4)

receiver of object dereference "changed_files" must be type of object but got "string"

(expression)